src/sys/wait.rs

```diff
@@ -93,6 +93,25 @@ pub enum WaitStatus {
     StillAlive
 }
 
+impl WaitStatus {
+    /// Extracts the PID from the WaitStatus if the status is not StillAlive.
+    pub fn pid(&self) -> Option<Pid> {
+        use self::WaitStatus::*;
+        match self {
```

Comment:
I'm not certain if this [StackOverflow comment](https://stackoverflow.com/questions/36590549/matching-on-a-reference-to-an-enum) is correct, but it would read nicer if you did `match *self` to get rid of the `&` on every match arm.